### DISCUSSION TITLE
[Feature Request] Full autogen step support

### DISCUSSION THREAD

**Discussion by jack5github** [TeamRizu/OutFox#348] [03 - Feature Requests]
Since Project Outfox's primary focus is to support many different game types with enough charts to make any of them viable, I definitely think it would help for the Autogen Steps setting to get some attention, for a lot of the available game types have step types that are basically the same as other game types, for example: Dance 4-key could have its steps transferred to Maniax, Techno, Popn and Kickbox. If that's not how Autogen Steps is meant to work, and it has some other purpose that I have yet to notice, then we need this functionality in the game anyway.

Edit: I've discovered that Autogen Steps requires you to begin editing a chart in order to generate steps for it. Isn't this a little counter intuitive? Ideally all charts should immediately be available and adapt to changes to real ones.

**Comment by Scraticus** [comment]
you can just enable the autogensteps option in the preferences, it'll gen them for you for every mode.

however, as many of these new modes were not created with dance in mind, (the autogen was made for dance) it can and will make some nasty decisions. we suggest using super shuffle with the autogen charts to allow them to distrobute the notes a bit fairer.
